pkg/utils/utils.go

```diff
@@ -53,6 +55,40 @@ func (u *utils) WriteFile(path string, data []byte) error {
 	return os.WriteFile(path, data, 0644)
 }
 
+func (u *utils) resolveImageStreamTagString(s string) (string, error) {
+	namespace, name, tag := parseImageStreamTagString(s)
+	if len(namespace) == 0 {
+		return "", fmt.Errorf("expected namespace/name:tag")
+	}
+	return u.resolveImageStreamTag(namespace, name, tag)
+}
+
+func parseImageStreamTagString(s string) (string, string, string) {
+	var namespace, nameAndTag string
+	parts := strings.SplitN(s, "/", 2)
+	switch len(parts) {
+	case 2:
+		namespace = parts[0]
+		nameAndTag = parts[1]
+	case 1:
+		nameAndTag = parts[0]
+	}
+	name, tag, _ := imageutil.SplitImageStreamTag(nameAndTag)
+	return namespace, name, tag
+}
+
+func (u *utils) resolveImageStreamTag(namespace, name, tag string) (string, error) {
+	imageStream, err := u.ImageClient.ImageStreams(namespace).Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
This is not a test please don't use `context.TODO()` you need to use a context with a timeout so the application will not get stuck.